packages/protocol-kit/tests/e2e/safeFactory.test.ts

```diff
@@ -192,8 +192,8 @@ describe('SafeProxyFactory', () => {
       const threshold = 2
       const safeAccountConfig: SafeAccountConfig = { owners, threshold }
       const safeDeploymentConfig: SafeDeploymentConfig = { saltNonce: '12345' }
-      const predictSafeProps: PredictSafeProps = { safeAccountConfig, safeDeploymentConfig }
-      const counterfactualSafeAddress = await safeFactory.predictSafeAddress(predictSafeProps)
+      const PredictedSafeProps: PredictedSafeProps = { safeAccountConfig, safeDeploymentConfig }
```

Comment:
The first should start with lower `p` `predictedSafeProps` to follow constant naming convention